.eslintrc.yml

```diff
@@ -8,7 +8,7 @@ globals:
   XRegExp: true
 extends: 'eslint:recommended'
 parserOptions:
-  ecmaVersion: 2016
+  ecmaVersion: 2019
```

Comment:
we are webpacking things for browser compatibility anyways. So our development version won't cause troubles